assets/js/base/components/notice-banner/style.scss

```diff
@@ -146,4 +146,23 @@
 	&.is-info {
 		@extend %info;
 	}
+	&.is-highlighted {
```

Comment:
Nice, that works, but the smooth scrolling does not, due to focusing the element. To fix this, I focused the element after a timeout of 500ms, which should be enough to scroll to the notice.

I don't think this removes the need to store which one is highlighted in state. Notices themselves have no knowledge of the order they appear in, and actually neither do we at the top level. In this PR we highlight the first error in the store, so that could be any notice (though it is likely to be in order from the top, given the order the containers are registered).